ch3/download_model_client.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+import joblib
+import pandas as pd
+import psycopg2
+from mlflow.client import MlflowClient
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import train_test_split
+
+# 0. set mlflow environments
+MLFLOW_TRACKING_URI = "http://localhost:5000"
+MLFLOW_S3_ENDPOINT_URL = "http://localhost:9000"
+
+os.environ["MLFLOW_S3_ENDPOINT_URL"] = MLFLOW_S3_ENDPOINT_URL
+os.environ["MLFLOW_TRACKING_URI"] = MLFLOW_TRACKING_URI
+os.environ["AWS_ACCESS_KEY_ID"] = "mystorage"
+os.environ["AWS_SECRET_ACCESS_KEY"] = "mystoragepw"
+
+# 1. get data
+db_connect = psycopg2.connect(
+    host="localhost",
+    database="postgres",
+    user="postgres",
+    password="mypassword",
+)
+df = pd.read_sql("SELECT * FROM iris_data ORDER BY id DESC LIMIT 10", db_connect)
+X = df.drop(["id", "target"], axis="columns")
+y = df["target"]
+X_train, X_valid, y_train, y_valid = train_test_split(
+    X,
+    y,
+    train_size=0.8,
+    random_seed=2022,
+)
+
+# 2. load model from mlflow
+client = MlflowClient(MLFLOW_TRACKING_URI)
+client.download_artifacts(run_id=sys.argv[1], path="sk_model.pkl", dst_path="./")
```

Comment:
```suggestion
client.download_artifacts(run_id=sys.argv[1], path="sk_model.pkl", dst_path="./")
```
sys 대신에 argparse 쓰는게 어떨까요?
그리고 이 파일에는 `import sys` 가  없네요

Reply:
@Kimdongui @Aiden-Jeon 음 이부분은 이전 동현님 PR 에서도 얘기했었는데요. `sys.argv` 이든 `argparse` 든 `os.getenv` 든 하나로 통일시키는게 좋을 것 같습니다. 하나 정해주세요 ㅋㅋ

Reply:
```bash
$ python download_model_client.py `RUN_ID`
```

이런 방식으로 사용하려고 했는데, `argparse`를 사용해도 비슷하게 구현 할 수 있을까요? 그렇다면 바꾸는게 더 직관적일 것 같긴합니다!

Reply:
도커에서 환경변수로 받아야 하며 값이 자주 변하지 않는 것들은 `MLFLOW_TRACKING_URI` 와 같은 것 들은 `os.getenv` 를 기반으로 하고
그 외에 값이 자주 변경되며 전달해줘야 하는 값들은 argparser 를 쓰면 좋을 것 같습니다.
이 방법에서는 도커를 실행할 때 CMD로 `["python", "download_model", "--run-id"]` 로 쓰고
`docker run 123123` 로 하는게 더 자연스러운 방법 같습니다.

Reply:
음.. 이 방식으로 구현하게되면 파일이 2개라 이미지도 2개가 나와서 조금 가벼운 방법으로 local에서 
```
$ python download_model_client.py --run-id `RUN_ID`
``` 
을 실행하는 방법으로 고민해봤는데 docker를 사용하는 방식으로 하는게 더 좋을까요?

Reply:
아 도커로 사용한다면 ! 입니다 ㅋㅋ
말씀해주신 방향으로 가면 될 것 같아요

Reply:
반영되었습니다 :)